app/models/theft_alert.rb

```diff
@@ -0,0 +1,79 @@
+class TheftAlert < ActiveRecord::Base
+  enum status: { pending: 0, active: 1, inactive: 2 }.freeze
+
+  validates :stolen_record,
+            :theft_alert_plan,
+            :status,
+            :creator,
+            presence: true
+
+  belongs_to :stolen_record
+  belongs_to :theft_alert_plan
+  belongs_to :payment
+  belongs_to :creator,
+             class_name: "User",
+             foreign_key: :user_id
+
+  scope :due_for_expiration, -> { active.where('"theft_alerts"."end_at" <= ?', Time.current) }
+
+  delegate :bike, to: :stolen_record
+
+  def begin!(kwargs)
+    start_time = Time.current.beginning_of_day
```

Comment:
I don't understand why we'd do `beginning_of_day` here rather than the current time. What is the intention of that?